### PR TITLE
Committing json and yaml files

### PR DESCRIPTION
**Pull request by ljs4x**
to complete assignment. unclear if the YAML is correct though...

### CODE REVIEW

File: fall2017/LukeSchwarz/swagger/CloudDev.yaml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        - in: body
+          name: body
+          description: Pet object that needs to be added to the store
+          required: true
+          schema:
+            $ref: '#/definitions/User'
+      responses:
+        '405':
+          description: Invalid input
+    put:
```
Comment (toddysm) [inline fall2017/LukeSchwarz/swagger/CloudDev.yaml]: There is no PUT for this path

File: fall2017/LukeSchwarz/swagger/CloudDev.yaml [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+  - name: user
+    description: Operations about user
+    externalDocs:
+      description: Find out more about our store
+      url: 'http://swagger.io'
+schemes:
+  - http
+paths:
+  /user:
+    post:
```
Comment (toddysm) [inline fall2017/LukeSchwarz/swagger/CloudDev.yaml]: You are missing GET method for this path

File: fall2017/LukeSchwarz/swagger/CloudDev.yaml [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+            $ref: '#/definitions/User'
+      responses:
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Pet not found
+        '405':
+          description: Validation exception
+  '/user/{id}/Task':
+    get:
```
Comment (toddysm) [inline fall2017/LukeSchwarz/swagger/CloudDev.yaml]: You need also POST for this path

File: fall2017/LukeSchwarz/swagger/CloudDev.yaml [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/User'
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Pet not found
+    post:
```
Comment (toddysm) [inline fall2017/LukeSchwarz/swagger/CloudDev.yaml]: There is no POST for this path. Did you mean PUT?

File: fall2017/LukeSchwarz/swagger/CloudDev.yaml [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+          required: true
+          type: integer
+          format: int64
+      responses:
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Pet not found
+      security: null
+definitions:
```
Comment (toddysm) [inline fall2017/LukeSchwarz/swagger/CloudDev.yaml]: You are missing path /user/{usrid}/task/{taskid}

**Comment by ljs4x** [comment]
This may work now. At least all the paths are there I think. I followed the model given on the assignment page on canvas because I lost my notes some how. but I had the paths and I/O diagram from class the other night so I think I pieced something together.

**Pull request merged**